src/is-opera.ts

```diff
@@ -0,0 +1,5 @@
+export = function isOpera(ua?: string): boolean {
+  ua = ua || window.navigator.userAgent;
+
+  return ua.indexOf("OPR/") !== -1 || ua.indexOf("Opera/") !== -1;
```

Comment:
Looks like the useragent on ios opera uses OPT instead of OPR. Can you add that handling as well?

Reply:
Sure thing! Added `OPT/` parsing, it seems that Opera on iOS is Webview-based, so returning true also for `isIosWebVew`

Reply:
Yeah, all 3rd party browsers on ios are actually Safari webviews.